bin/dbsync

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env node
+
+var path   = require('path');
+var fs     = require('fs');
+var coffee = require('coffee-script');
+var src    = path.join(path.dirname(fs.realpathSync(__filename)), '../src');
+
```

Comment:
Well the lint file can be as strict or loose as you want. But we should make some restrictions on outsiders.